en/controllers.rst

```diff
@@ -670,8 +670,8 @@ Other Useful Methods
             array('limit' => 5, 'order' => 'Article.created DESC')
         );
 
-        $this->loadModel('User', 2);
-        $user = $this->User->read();
+        $this->loadModel('User');
```

Comment:
I disagree because it *still* shows how load model works, using the 2nd param is a bad practice none the less.